.github/workflows/get-stalkerware-indicators.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Sync with IOC stalkerware indicators
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```

Comment:
Does it make sense to update the IOC list on PR branches to main?
Maybe remove the `pull_request` `on` item, and add a `workflow_dispatch` item so that it can be manually triggered in the Actions tab